plugins/OrchidChangelog/src/main/kotlin/com/eden/orchid/changelog/publication/GithubReleasesPublisher.kt

```diff
@@ -0,0 +1,91 @@
+package com.eden.orchid.changelog.publication
+
+import com.caseyjbrooks.clog.Clog
+import com.eden.orchid.api.OrchidContext
+import com.eden.orchid.api.options.annotations.Description
+import com.eden.orchid.api.publication.OrchidPublisher
+import com.eden.orchid.changelog.model.ChangelogModel
+import okhttp3.MediaType
+import okhttp3.Request
+import okhttp3.RequestBody
+import okhttp3.Response
+import okhttp3.OkHttpClient
+import org.json.JSONObject
+import java.text.SimpleDateFormat
+import java.time.Duration
+import java.time.Instant
+import javax.inject.Inject
+
+@Description("Create releases directly on Github.", name = "GithubReleases")
+class GithubReleasesPublisher
+@Inject
+constructor(
+        context: OrchidContext,
+        val model: ChangelogModel,
+        val client: OkHttpClient
+) : OrchidPublisher(context, "githubReleases", 100) {
+
+    @Description("A GitHub Personal Access Token.")
+    lateinit var githubToken: String
```

Comment:
The `githubToken` is sensitive data that is already set up to be passed as a command-line arg. You can inject this property in the constructor instead of having it be an option, the same as being done with [GithubPagesPublisher](https://github.com/JavaEden/Orchid/blob/dev/OrchidCore/src/main/java/com/eden/orchid/impl/publication/GithubPagesPublisher.java#L131).